scripts_common.py

```diff
@@ -84,6 +85,8 @@ def get_provenance_metadata(file, runcmd):
         runcmd: the command used to run the file (with any arguments)
     """
 
+    prepend = f"Created by {os.environ['USER']}, on {datetime.now().strftime('%Y-%m-%d')}, using " 
```

Comment:
> os.environ['USER']

Is there a way to put the actual name in parentheses after this? Maybe using `git config --global user.name` if it exists?

Reply:
Ok done :)